lms/djangoapps/instructor/views/api.py

```diff
@@ -2165,23 +2165,35 @@ def list_background_email_tasks(request, course_id):
     return JsonResponse(response_payload)
 
 
-@require_POST
```

Comment:
It's confusing that this is a POST request in the first place given that it is not sending any data in the POST body and not making any modifications.  But that change was made in https://github.com/openedx/edx-platform/pull/12719 for reasons that are unclear to us now so we'll leave it as is for now and we can re-visit if/when we want to revamp the instructor APIs further.

This will at-least make these APIs all visible in our openapi spec and make the APIs usable by other token types than just session.

(Not something you need to do anything about, just leaving a not for future investigators.)